tests/e2e/README.md

```diff
@@ -1,117 +1,144 @@
+  --num-nodes ${NUM_NODES} \
+  --enable-kubernetes-alpha \
+  --no-enable-legacy-authorization
+ ```
 
-istio.VERSION can be updated by [updateVersion.sh](../../updateVersion.sh).
-Look at [Integration Test](https://github.com/istio/istio/tree/master/tests#updateversionsh) for more information.
+ - `CLUSTER_NAME`: Whatever suits your fancy, 'istio-e2e' is a good choice.
+ - `ZONE`: 'us-central-f' is a good value to use.
+ - `PROJECT_NAME`: is the name of the GCP project that will house the cluster. You get a project by visiting [GCP](https://console.cloud.google.com).
+ - `CLUSTER_VERSION`: 1.7.3 or later.
+ - `MACHINE_TYPE`: Use 'n1-standard-4'
+ - `NUM_NODES`: Use 3.
+ - `no-enable-legacy-authorization`: Optional, needed if you want to test RBAC.
 
-If not specify `namespace`, a randomly namespace would be generated for each test.
+You must set your default compute service account to include:
+roles/container.admin (Kubernetes Engine Admin)
+Editor (on by default)
 
-`log_provider` and `project_id` must both be specified if one wishes to collect cluster logs.
+To set this, navigate to the IAM section of the Cloud Console and find your default GCE/GKE service account in the
+following form: projectNumber-compute@developer.gserviceaccount.com: by default it should just have the Editor role.
+Then in the Roles drop-down list for that account, find the Kubernetes Engine group and select the role Kubernetes
+Engine Admin. The Roles listing for your account will change to Multiple.
 
-### For all the following example, you always need to add:
-* `--auth_enable` if you want to include auth
-* `--cluster_wide` if you want to run the cluster wide installation and tests
-* `--use_initializer` if you want to do transparent sidecar injection
+## Step 2: Get cluster credentials
+```
+gcloud container clusters get-credentials ${CLUSTER_NAME} --zone ${ZONE} --project ${PROJECT_NAME}
+```
 
-### Example
-From the repo checkout root directory
+## Step 3: Create Clusterrolebinding
+```
+kubectl create clusterrolebinding prow-cluster-admin-binding    --clusterrole=cluster-admin    --user="your cluster user name(for example youralias@google.com)"
+```
 
-* Run tests with the latest stable version of istio according to istio.VERSION :
+## Step 4: Export test script variables
 
-`tests/e2e.sh --auth_enable`
+**Option 1:** Already committed changes to istio/istio master branch
+NOTE: SHA used for GIT_SHA is one that is already committed on istio/istio. You can pick any SHA you want.
+```
+export HUB="gcr.io/istio-testing"
+export GIT_SHA="d0142e1afe41c18917018e2fa85ab37254f7e0ca"
+```
 
-* Test commit in pilot repo, SHA:"dc738396fd21ab9779853635dd22693d9dd3f78a":
+**Option 2:** Testing local changes
 
-`tests/e2e.sh --pilot_hub=gcr.io/istio-testing --pilot_tag=dc738396fd21ab9779853635dd22693d9dd3f78a --istioctl_url=https://storage.googleapis.com/istio-artifacts/dc738396fd21ab9779853635dd22693d9dd3f78a/artifacts/istioctl --auth_enable`
```

Comment:
unless the test now use the locally built istioctl this was an important piece of information (istioctl_url)

Reply:
Yes, I have not removed this. The below step has it:
```
--istioctl_url "https://storage.googleapis.com/istio-artifacts/pilot/${GIT_SHA}/artifacts/istioctl"   \
```